### PR TITLE
update to 1.13.2

### PR DESCRIPTION
**Pull request by null-null-null**
@firefly-ui/owners

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,11 +27,12 @@
...
     "ember-cli-uglify": "^1.0.1",
-    "ember-data": "1.0.0-beta.17",
+    "ember-data": "1.13.2",
+    "ember-disable-prototype-extensions": "^1.0.0",
     "ember-disable-proxy-controllers": "^0.7.0",
     "ember-export-application-global": "^1.0.2",
-    "ember-disable-prototype-extensions": "^1.0.0",
-    "ember-try": "0.0.4"
+    "ember-try": "0.0.4",
+    "ember-validations": "2.0.0-alpha.3"
```
Comment (twokul) [inline package.json]: that doesn't look like an update ;) can you submit a different PR?

Comment (null-null-null) [inline package.json]: Do you mean that the version wasn't bumped? Just bumped it. Aside from that, there are no deprecations in this component (aside from one caused by ember-validations, but that's only a devDepedency), so the only update was to bower.json and package.json to get them using 1.13.2.

Comment (twokul) [inline package.json]: no I mean additional dependency was added which we didn't agree upon

Comment (null-null-null) [inline package.json]: Removed the devDep. Let me know if there's anything else.
